### PR TITLE
[kmac] Countermeasures asset in Hjson

### PR DESCRIPTION
**Pull request by eunchan**
What this commit does:
    Added countermeasures implemented in KMAC into Hjson

Why is it valuable:
    The `kmac.hjson` now  lists the countermeasures to help
    understanding of possible attacks and protections.

Co-Authored-by: Michael Tempelmeier <michael.tempelmeier@gi-de.com>

### CODE REVIEW

File: hw/ip/kmac/data/kmac.hjson [hunk 1]
```diff
@@ -152,6 +152,21 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "KEY.SCA"
```
Comment (cdgori) [inline hw/ip/kmac/data/kmac.hjson:155]: I think for AES we decided this should be KEY.MASKING

File: hw/ip/kmac/data/kmac.hjson [hunk 2]
```diff
@@ -152,6 +152,21 @@
...
+      desc: '''Data storage and secret key are two share to guard against 1st
+        order attach'''
+    }
+    { name: "CONFIG.SHADOW"
+      desc: "Shadowed CFG register."
+    }
+    { name: "FSM.SPARSE"
+      desc: "FSMs in KMAC are sparsely encoded."
+    }
+    // TODO: LFSR
```
Comment (cdgori) [inline hw/ip/kmac/data/kmac.hjson:165]: These will be done before D2S review?

**Review by cdgori — APPROVED** [review]
Please change to KEY.MASKING, otherwise LGTM (more to come I assume...)

File: hw/ip/kmac/data/kmac.hjson (hunk 2 above)
Comment (eunchan) [reply hw/ip/kmac/data/kmac.hjson:165]: yep

**Review by tjaychen — APPROVED** [review]

**Comment by eunchan** [comment]
As far as I know, it is OK to combine them to one category. Splitting between the assets is also fine.

**Pull request merged**